sdk/src/client/stronghold/mod.rs

```diff
@@ -513,6 +516,11 @@ impl StrongholdAdapter {
 
         Ok(())
     }
+
+    /// Acquire the stronghold lock.
+    pub async fn get_stronghold(&self) -> MutexGuard<'_, Stronghold> {
```

Comment:
We don't usually put a `get_` prefix on our getters.
Also, `stronghold.get_stronghold()` may be a bit weird, I'd suggest just `inner` but let's see what the other think

Reply:
I think `inner` also works fine.

Reply:
Id like to see .inner() instead as well